module/dice/damage-roll.mjs

```diff
@@ -302,7 +302,9 @@ export default class DamageRoll extends BasicRoll {
     );
     const DialogClass = this.DefaultConfigurationDialog;
```

Comment:
```suggestion
    const DialogClass = this.constructor.DefaultConfigurationDialog;
```

Reply:
This one's already a static method (which is what confused me and caused me to forget that `configureDialog` in `D20Roll` wasn't).